### DISCUSSION TITLE
JetPack 5.1.1, L4T 35.3.1

### DISCUSSION THREAD

**Discussion by noob502** [OE4T/meta-tegra#1505] [Q&A]
Is thre any branch for  JetPack 5.1.1, L4T 35.3.1 ? The kirkstone support only for L4T 35.4.1 ? According to [Ridgerun](https://developer.ridgerun.com/wiki/index.php/Yocto_Support_for_NVIDIA_Jetson_Platforms_-_Setting_up_Yocto) kirkstone supported 35.3.1.

**Comment by madisongh** [comment]
It probably did, back when they first wrote that page. You could let them know that it needs updating.

[This wiki page](https://github.com/OE4T/meta-tegra/wiki/Which-branch-should-I-use%3F) lists the branches and their contents.  None of them are currently on R35.3.1; the kirkstone branch was updated to R35.4.1 back in September 2023, and will soon be updated to R35.5.0. If you really need to go back to R35.3.1 for some reason, you could check out the kirkstone branch at commit dd0d91ad1ac0f0654aed6cce143ed8fc26719196, just before the R35.4.1 update was brought in.
